### PR TITLE
Added info about brute force protection

### PR DESCRIPTION
**Pull request by lzap**
New feature is in.

### CODE REVIEW

**Comment by ekohl** [comment]
I assume users of foreman_memcache need to remove keys from memcache and this won't apply.

**Comment by lzap** [comment]
Good point, added a remark about this.

File: _includes/manuals/1.16/4.1.1_authentication.md [hunk 1]
```diff
@@ -227,6 +227,14 @@ When using Active Directory, please be aware that users will be able to log in f
 
 To change this password expiry period, see [Microsoft KB906305](https://support.microsoft.com/en-us/kb/906305) for the necessary registry change.
 
+#### Brute-force protection
+
+Foreman allows only 30 failed attempts in the last 5 minutes per one IP address by default. Any subsequent login attempts are not allowed and error message is shown: "Too many tries, please try again in a few minutes." If this is triggered by accident, the silent period can be removed by deleting failed login cache entries:
+
+    find /usr/share/foreman/tmp/cache -name failed_login\*
+
+When using memcache plugin, delete the same keys in the database instead. Amount of attempts and silent period is not configurable.
```
Comment (ekohl) [inline _includes/manuals/1.16/4.1.1_authentication.md]: using *the* memcache plugin. I'm also not sure we should refer to memcache as the database (IMHO that's still PostgreSQL or MySQL). Another difficulty is that memcache itself doesn't really have a mechanism to delete keys starting with some string.

Comment (lzap) [reply _includes/manuals/1.16/4.1.1_authentication.md]: Ok how about this.

**Review by ekohl — APPROVED** [review]

**Comment by mmoll** [comment]
is this a 1.16 feature? or 1.17?

**Comment by ares** [comment]
1.17 based on related foreman issue

**Comment by lzap** [comment]
Oh right, I should have moved this into some TODO branch, do we have any @xprazak2 ?

**Comment by xprazak2** [comment]
My wip docs for [1.17](https://github.com/theforeman/theforeman.org/pull/983)

**Comment by GregSutcliffe** [comment]
I presume this can be merged now that 1.17 is out, @mmoll / @xprazak2 / @lzap?

**Comment by mmoll** [comment]
it just needs to get changed to the 1.17 file

**Comment by lzap** [comment]
I do see it there, so closing:

https://github.com/theforeman/theforeman.org/blob/gh-pages/_includes/manuals/1.17/4.1.1_authentication.md

**Pull request closed without merging**